code-examples/quarkus-registry-kafka-quickstart/processor/src/main/resources/application.properties

```diff
@@ -0,0 +1,26 @@
+%dev.quarkus.http.port=8081
+
+# Configure the incoming `quote-requests` Kafka topic
+mp.messaging.incoming.requests.connector=smallrye-kafka
+mp.messaging.incoming.requests.topic=quote-requests
+mp.messaging.incoming.requests.auto.offset.reset=earliest
+mp.messaging.incoming.requests.value.deserializer=io.apicurio.registry.serde.avro.AvroKafkaDeserializer
+mp.messaging.incoming.requests.apicurio.registry.use-specific-avro-reader=true
+
+%prod.mp.messaging.incoming.requests.apicurio.auth.service.token.endpoint=${OAUTH_TOKEN_ENDPOINT_URI}
+%prod.mp.messaging.incoming.requests.apicurio.auth.client.id=${CLIENT_ID}
+%prod.mp.messaging.incoming.requests.apicurio.auth.client.secret=${CLIENT_SECRET}
+
+
+
+# Configure the outgoing `quotes` Kafka topic
+mp.messaging.outgoing.quotes.apicurio.registry.auto-register=true
+mp.messaging.outgoing.quotes.connector=smallrye-kafka
+mp.messaging.outgoing.quotes.value.serializer=io.apicurio.registry.serde.avro.AvroKafkaSerializer
+
+%prod.mp.messaging.outgoing.quotes.apicurio.auth.service.token.endpoint=${OAUTH_TOKEN_ENDPOINT_URI}
```

Comment:
What is the idea behind prod profile?

Reply:
The idea is to avoid the need of configuring auth for dev or test profiles. When running with prod profile the application will not start a devservice so registry + kafka become a prerequisite. In dev and test it's starting kafka and registry devservices without auth so there's no need to configure this.

Reply:
Ok. So running locally this would not connect to remove service registry (like Kafka guide does) which is fine. 
And then prod profile can be something we build with kubernetes plugin and push to OpenShift cluster?

Apologies for asking those questions - I'm actually wanting to know what is the plan for the narrative of the guide

Reply:
Yeah, that's exactly the idea behind this.

Reply:
I like it - then we can create another profile - SBO - that will inject most of the values we have.